packages/ui/src/theme/css/component/authenticator.scss

```
@@ -41,7 +41,10 @@
   background-color: var(--amplify-colors-background-primary);
   box-shadow: var(--amplify-shadows-small);
   place-self: center;
-  width: calc(var(--amplify-breakpoints-values-small) * 1rem);
+  width: 460px;
+  @media (max-width: 30em) {
+    width: 100%;
+  }
```

Let's say we set this to 460px, and then reduce less then < 30em ? Also, `--amplify-breakpoints-values-small` doesn't exist, not sure if it's not being generated correctly 🤷

---

I believe we should use the same units here (`rem` should be more accurate to the device text size).

```suggestion
  width: 30rem;
  @media (max-width: 30rem) {
    width: 100%;
  }
```

This was previously set to 36rem (#362), but I'll defer on your judgement on which value looks better.

---

OK! I changed it to 30rem! I think it looks good at that.